### PR TITLE
feat: [CDS-71892]:terraform-steps-fix

### PR DESCRIPTION
**Pull request by litvinovstas**
### Summary
Fixed issue with removing configuration/cloudConfiguration properties from terraform steps /plan/apply/destroy

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/70-pipeline/components/PipelineStudio/RightDrawer/RightDrawer.tsx [hunk 1]
```diff
@@ -69,6 +69,8 @@ export const FullscreenDrawers: DrawerTypes[] = [
   DrawerTypes.PolicySets
 ]
 
+export const TerraFormSteps: string[] = [StepType.TerraformApply, StepType.TerraformPlan, StepType.TerraformDestroy]
```
Comment (PratyushGar) [inline src/modules/70-pipeline/components/PipelineStudio/RightDrawer/RightDrawer.tsx:72]:
```suggestion
export const TerraformSteps: string[] = [StepType.TerraformApply, StepType.TerraformPlan, StepType.TerraformDestroy]
```

**Review by PratyushGar — DISMISSED** [review]

File: src/modules/70-pipeline/components/PipelineStudio/RightDrawer/RightDrawer.tsx [hunk 2]
```diff
@@ -69,6 +69,8 @@ export const FullscreenDrawers: DrawerTypes[] = [
   DrawerTypes.PolicySets
 ]
 
+export const TerraformSteps: string[] = [StepType.TerraformApply, StepType.TerraformPlan, StepType.TerraformDestroy]
```
Comment (rameshx) [inline src/modules/70-pipeline/components/PipelineStudio/RightDrawer/RightDrawer.tsx:72]: can be moved to a constants/util file

**Review by ishantmehtaharness — DISMISSED** [review]
looks good to me

**Review by ishantmehtaharness — APPROVED** [review]
looks good

**Review by rameshx — APPROVED** [review]

**Comment by litvinovstas** [comment]
Ticket for writing testcases for RightDrawer.tsx -> https://harness.atlassian.net/browse/CDS-71247

**Review by YounixIO — APPROVED** [review]

**Pull request merged**